src/helpers/fix.js

```diff
@@ -278,6 +367,7 @@ export const REPLACE_PREFIXES = [
 ];
 
 // If placed in config.js testing needs envs
```

Comment:
Create file constants.js
and place const exports in there

Reply:
Created a separate file for fix-constants

[64ab692](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/64ab692f906645b369e9122173a1420a5b211f82)